### ISSUE TITLE
PPL异常

### ISSUE THREAD

**Issue by 2020zyc** [zai-org/GLM-5#32]
<img width="733" height="282" alt="Image" src="https://github.com/user-attachments/assets/fbfd905e-e383-4f99-a3d6-cabb2adf1a7d" />

1. 测试如第一个红框的中文，使用generate函数进行输出，看起来不太通顺（第二个红框）
2. 还是第一个红框的中文，使用model(input_ids)获得outputs，通过outputs里的logits与input_ids（label）一起计算ppl，达到170多。相同的方法，之前测试deepseek r1才3点几。
3. 测试代码如下图

<img width="832" height="923" alt="Image" src="https://github.com/user-attachments/assets/e0371c0a-8b03-42ac-9fba-938e42678288" />

请问以上现象是正常的吗，第一个问题是需要template？第二个问题，不管加不加template，都是100多，这太高了？
麻烦帮忙看有一下，谢谢！@zRzRzRzRzRzRzR

**Comment by 2020zyc** [comment]
@zRzRzRzRzRzRzR 打扰了 请问有空复现吗？或者需要我补充什么信息吗？
torch 2.8.0 
transformers 5.2或者5.3都一样；
昨天又尝试使用deepseek的fp8 to bf16脚本，将fp8权重转到bf16，测得PPL结果也是100多；
但是使用VLLM加载glm-fp8模型测PPL，相同wiki数据和评测方法，得到的ppl是2点几；
那是否说明是transformers+glm5有问题？

**Comment by JaredforReal** [comment]
@2020zyc GLM5+transformers的实现确实还存在一些问题，我们正在排查修复～感谢您的理解！

**Comment by ArthurZucker** [comment]
会看看!
